apps/website/src/pages/docs/expandableblock.mdx

```diff
@@ -9,67 +9,83 @@ group: core
+</LiveExample>
 
-The child elements don’t reflect the column titles. They are expected to retain the same properties as the parent row (in the example below, the owner). It’s unnecessary to repeat the properties for the child items because of this, so do not include them in the expanded row area.
+An expandable block could display status, an icon, or other helpful information at a glance right aligned within the block header.
 
-Child items are not interchangeable between rows, e.g. you cannot drag a child item from row A and drag it in row B like you would move files to a folder.
+### Small
 
-### Expandable blocks
+<LiveExample src='ExpandableBlock.small.tsx'>
+  <AllExamples.ExpandableBlockSmallExample client:load />
+</LiveExample>
 
-The main block has a label reflecting its content. On click, it expands to reveal more content.
+An expandable block can be one of two sizes: default and small
 
-An expandable block could display status, a total count, or other helpful information at a glance right aligned within the block header. You may also optionally display descriptor text underneath the title.
+### Borderless
 
-Expandable blocks don’t always need to be sorted alphabetically, unlike an expandable row which would be sorted based on the table column sort. You may optionally allow expandable blocks to be re-ordered by the user.
+<LiveExample src='ExpandableBlock.borderless.tsx'>
+  <AllExamples.ExpandableBlockBorderlessExample client:load />
+</LiveExample>
 
-The visual appearance of an expandable block is different from an expandable row. An expandable block can not be selected / checked like an expandable row can, therefore it is visually styled differently.
+The borderless expandable block is useful in tighter areas where using a default expandable block would visibly cause too many blocks.
 
 ## Usage
 
-### Expandable rows
+- Expandable blocks are used to progressively disclose information. [Read this article to learn more about progressive disclosure.](https://www.nngroup.com/articles/progressive-disclosure/)
 
-Expandable rows are used in tables. The layout inside expanded rows varies according to requirements. Child rows can be selectable and be an entry point to more content (ex.: downloadable files), or only contain metadata about the parent file. If at least one row within a table is expandable, all rows should follow the same pattern. Avoid mixing regular table rows with expandable rows.
+- Expandable blocks should not be used with flattened hierarchy or information panels.
 
-### Expandable blocks
+### Forms
 
-Expandable blocks are used in pages to divide up content in sections, in forms for example. If each section has many fields, using expandable blocks will help alleviate the UI and prevent the user from scrolling endlessly to get to the section they are looking for.
+<LiveExample src='ExpandableBlock.form.tsx'>
+  <AllExamples.ExpandableBlockFormExample client:load />
+</LiveExample>
```

Comment:
Let me know if this form example needs to be different or longer

Reply:
When multiple blocks are opened, I can't scroll to the top: 
![image](https://user-images.githubusercontent.com/37936169/230214744-8fb7d269-272c-4ff1-8dfc-b154a11c16e2.png)

It's a good example, just needs this bug fixed.

Reply:
Fixed